architecture-compose/src/main/kotlin/com/splendo/kaluga/architecture/compose/navigation/RouteController.kt

```diff
@@ -58,7 +61,8 @@ sealed interface RouteController {
 
 /**
  * Abstract [RouteController] that navigates on a [NavHostController] provided by a [Provider]
- * @param provider The [Provider] of the [NavHostController]
+ * @param Provider a type of class that can provide a [NavHostController]
```

Comment:
Where is this param called Provider?

Reply:
In Kdoc, you should also use @param to provide context to Generic constraints
https://kotlinlang.org/docs/kotlin-doc.html#block-tags